environment/cpu/src/cloud/benchflow/monitors/cpu.go

```diff
@@ -82,6 +82,7 @@ func monitorStats(container Container) {
 				return
 			default:
 				stat := (<-container.statsChannel)
+				if(stat == nil) {panic("Received Nil from container")}
```

Comment:
Can this happen during the execution or only when the execution starts? I guess the first, right? If that is the case I would say we need a better way to handle this error, by reporting the status to the client.